.github/CODEOWNERS

```diff
@@ -1,5 +1,7 @@
 # @1024pix/devcomp owns any file in the `/api/src/devcomp/` directory in the root of your repository and any of its subdirectories.
 /api/src/devcomp/ @1024pix/team-devcomp
+!/api/src/devcomp/infrastructure/datasources/learning-content/modules/*.json
```

Comment:
Pas certain de la syntaxe, j'ai trouvé ça là https://dev.to/eunice-js/a-comprehensive-guide-to-codeowners-in-github-22ga